README.md

```diff
@@ -6,5 +6,19 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+
+  Rails is a web framework to help build the back end of web apps. It provides default structures for a database, web service and web pages.
+
+
 1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+
+  Image inserted in submissions directory.
+
 1. Describe the flow of a PATCH request as it goes through the app.
+
+    1. client = request to server to change a field
+    2. server = relays request to the controller
+    3. controller = sends the request to the model
```

Comment:
Not quite. The controller never shares the request with the model - it just invokes the appropriate methods on the model to accomplish the request.